### PR TITLE
Create author profile

### PR DESCRIPTION
**Pull request by PaulGrishunin**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ZPoreba) [inline .idea/vcs.xml:1]: do not push .idea folder,  I didn't catch it and accepted some previous pr with .idea. You can remove whole .idea directory and add it to .gitignore

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (ZPoreba) [inline .idea/.gitignore:1]: do not push files from .idea to directory, add in file BooksReviewPlatform/client/.gitignore .idea/

File: client/src/AuthorProfile/AuthorProfile.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    loadAuthorData() {
+        let authorData = Object.keys(data).filter(item => { return data[item].id === this.state.authorId })[0];
+        authorData = data[authorData];
+
+        if(authorData.photo.length === 0) {
+            authorData.photo = defaultAuthor;
+        }
+        this.setState({authorData: authorData})
+console.log(authorData.books);
```
Comment (ZPoreba) [inline client/src/AuthorProfile/AuthorProfile.js:40]: delete consol log

File: .idea/vcs.xml (hunk 1 above)
Comment (PaulGrishunin) [reply .idea/vcs.xml:1]: ok

**Comment by SzymonSmykala** [comment]
@PaulGrishunin what about that?

**Pull request merged**